services/gateway/endpoints/src/main/java/org/eclipse/ditto/services/gateway/endpoints/routes/AbstractRoute.java

```diff
@@ -61,6 +64,9 @@
  */
 public abstract class AbstractRoute {
 
+    // TODO YC: refactor config key.
```

Comment:
what about that TODO? Is that for after the configuration refactoring?

Reply:
Yes.